src/abstractdataframe/abstractdataframe.jl

```diff
@@ -2173,3 +2173,209 @@ Base.getindex(::AbstractDataFrame, ::Union{Symbol, Integer, AbstractString}) =
 
 Base.setindex!(::AbstractDataFrame, ::Any, ::Union{Symbol, Integer, AbstractString}) =
     throw(ArgumentError("syntax df[column] is not supported use df[!, column] instead"))
+
+# insertcols!
```

Comment:
Can you use two separate commits, one just moving the code to the new place and the next one making actual changes? That will help reviewing and if we  keep both when merging it will also make the history cleaner.

Reply:
OK - so for now I will not move this code to abstractdataframe.jl but keep it in dataframe.jl (and move it later). The reason is that we squash merge PRs anyway.